taxonomies/packaging_recycling.txt

```diff
@@ -74,7 +74,11 @@ packaging_materials:en: en:plastics
 <en:recycle
 en:return to store, deposit
 fr:consigné, consigne, rapporter au magasin, retourner au magasin
-# nl:recycle-als-statiegeld is already coer
+# nl:recycle-als-statiegeld is already covered
+
+#uk recycle instruction for films
+<en:recycle
+en:don't recycle at home (recycle with bags at large supermarket)
```

Comment:
I live in Edinburgh, UK and can confirm that this message appears fairly frequently on plastic product packaging.

A trustworthy source for some of the other symbols/messages that appear on packaging can be found here: https://www.which.co.uk/reviews/shopping-sustainably/article/how-to-recycle-in-the-uk-ajwEz4p63Qs6#packaging-symbols-explained

Reply:
Clarification of the wording of `can confirm`: I mean I _already_ know that it does; frequently have had 5+ items set aside with this on it.  Not future-tense `I am able to go and check whether it is true`.

Reply:
Here is how it would typically appear:

![recycle-at-store](https://github.com/user-attachments/assets/cf9e0d3c-ab60-4db5-b9c9-53a0063eaa67)